languages/rust/exercises/concept/tuples/.docs/introduction.md

```diff
@@ -0,0 +1,83 @@
+# Tuples
```

Comment:
I really like this introduction document. You've managed to keep things very concise while still being very clear.

Reply:
I like this introduction too. I think you explained things very clearly.